samples/packages/pulumi-typescript/src/cdf.manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://openfabr.github.io/cdf/references/typescript/2.0.0/manifest.schema.json",
```

Comment:
Have you validated it against this schema? Better do it to avoid setting a bad example :)

Reply:
checked. ✅